Update Equinix configuration file (#47)
harvester/ipxe-examples: equinix/README.md

```
#!ipxe
dhcp
iflinkwait -t 5000 && echo Detected link on ${ifname} 
set version master # change to a specific version for production installation

> set base https://releases.rancher.com/harvester/${version}  # currently ipxe on Equinix is not supported https://release.rancher.com, you can build your own web server or use other methods to download the relevant dependencies.

Reviewer: ```suggestion
set base https://releases.rancher.com/harvester/${version}  # ipxe on Equinix currently does not support `https://release.rancher.com`, you can build your own web server or use other methods to download related artifacts.
```

The `harvester.install.config_url=https://metadata.platformequinix.com/userdata` parameter tells the installer we want to fetch the Harvester configuration from this URL, which contains the userdata specified by the user when creating nodes. The harvester configuration contains sensitive credentials. We can prevent those credentials from leaking by using [userdata](https://metal.equinix.com/developers/docs/servers/user-data/) that can only be seen by provisioning nodes.

> **Note:** Currently ipxe on Equinix is not supported `https://release.rancher.com`, you can build your own web server or use other methods to download the relevant dependencies.

Reviewer: ```suggestion
**Note:** Due to a known [certificate issue](https://github.com/harvester/harvester/issues/2226), ipxe on Equinix currently does not support `https://release.rancher.com`, you can build your own web server or use other methods to download related artifacts.
```

    ```yaml
    #cloud-config
    token: token           # replace with a desired token
    os:
      ssh_authorized_keys:
      - ssh-rsa ...        # replace with your public key
      password: p@ssword   # replace with a your password
      ntp_servers:
      - 0.suse.pool.ntp.org
      - 1.suse.pool.ntp.org
    install:
      mode: create
      device: /dev/sda
      iso_url: https://releases.rancher.com/harvester/master/harvester-master-amd64.iso
      tty: ttyS1,115200n8

>       vip: <server_public_ip> # Randomly fill in the IP first or reinstall when the Public IP is known

Reviewer: @ibrokethecloud is this the best method we can do for now of the Equinix platform installation?

Reviewer 2: @guangbochen we can create an elastic IP in advance and attach it to the instance, and then use the same in vip.

This is what my sample terraform module does.

    ```yaml
    #cloud-config
    token: token           # replace with a desired token
    os:
      ssh_authorized_keys:
      - ssh-rsa ...        # replace with your public key
      password: p@ssword   # replace with a your password
      ntp_servers:
      - 0.suse.pool.ntp.org
      - 1.suse.pool.ntp.org
    install:
      mode: create
      device: /dev/sda
      iso_url: https://releases.rancher.com/harvester/master/harvester-master-amd64.iso
      tty: ttyS1,115200n8

>       vip: <server_public_ip> # use Elastic IPs or reinstall when the Public IP is known

Reviewer: ```suggestion
      vip: <server_public_ip> # use Elastic IP or reinstall when the public IP is known
```

      vip_mode: static
    ```
    - The [userdata file](./userdata-create.yaml) is included in the source and the user can modify it according to any need.
    - The Harvester installer doesn't require the `#cloud-config` line at the beginning, but Equinix Metal validates if the userdata contains it.

>     - The `vip` parameter is required to be filled.

Reviewer: ```suggestion
    - The `vip` parameter is required.
```

>   - To configure Elastic IPs for Public IPv4, enable `Configure IPs`, select `Deploy from your subnet` and select that the Subnet Size must be less than or equal to 31 (optional).

Reviewer: ```suggestion
  - (optional)[configure Elastic IP](#configure-elastic-ip) and use it as the `vip`.
```

- `install.mode`: Should be `join`.

**NOTE:** the user can create multiple servers to join a cluster with the same userdata at once.

> ### Request Elastic IPs (optional)
> - Select `IPs & Networks` from the menu bar, and then click `IP`.
> - Click `Request IP Addresses`.
> - Select `Public IPv4` item for deployment type.
> - Select the `Location` to be new.
> - The quantity must be selected to be **less than or equal to 31**.
> - Click `Submit Request` and wait until the new IP.

Reviewer: ```suggestion
### Configure Elastic IP
- Select `IPs & Networks` from the menu bar, and then click `IP`.
- Click `Request IP Addresses`.
- Select `Public IPv4` item for deployment type.
- Select the `Location` to be new.
- The quantity should be **less than or equal to 31**.
- Click `Submit Request` and wait to get the elastic IP.
```
